index.js

```diff
@@ -110,7 +110,7 @@ cs.get.rgb = function (string) {
 			return null;
 		}
 
-		rgb = colorNames[match[1]];
+		rgb = [...colorNames[match[1]]];
```

Comment:
```suggestion
		rgb = colorNames[match[1]].slice();
```

This is about 30% faster than spreads on at least v8.

Reply:
I know, but when running the tests it wanted me to change it to spreads,

Reply:
sure

Reply:
Ugh, that's an annoying rule. Maybe updating XO to the latest version would fix it but that's for another day.